distributed/distributed.yaml

```diff
@@ -71,6 +71,10 @@ distributed:
       pause: 0.80  # fraction at which we pause worker threads
       terminate: 0.95  # fraction at which we terminate the worker
 
+  nanny:
+    preload: []
```

Comment:
Most entries have a comment describing them

Reply:
Thanks for flagging this.

I've gone ahead and added brief comments that mostly just point to documentation (I don't think that we can reasonably describe this in one line).  Ben and I are planning to play with larger descriptions for documentation later this week though.